orangecontrib/timeseries/widgets/owlinechart.py

```diff
@@ -405,40 +411,47 @@ def setType(self, ax, type):
         ''' % locals())
 
 
-class LineChartContextHandler(DomainContextHandler):
+class LineChartContextHandler(ContextHandler):
     """
     Context handler of line chart. The specifics of this widget is
     that `attrs` variable is list of lists and it is not handled with
     the DomainContextHandler.
     """
+    def open_context(self, widget, domain_fetures):
+        if domain_fetures[0] is None:
```

Comment:
Could this ever be True? As far as I see, the context is opened only when data is not None.

Reply:
Thank you. It was left there from the previous implementation which was different. Reimplementing open_context is not required anymore. I removed this funciton.